docs/site/tutorials/a_swift_tour.ipynb

```diff
@@ -1656,7 +1700,7 @@
         "\n",
         "3.  Changing the value of properties defined by the superclass. Any additional setup work that uses methods, getters, or setters can also be done at this point.\n",
         "\n",
-        "If you don’t need to compute the property but still need to provide code that is run before and after setting a new value, use `willSet` and `didSet`. The code you provide is run any time the value changes outside of an initializer. For example, the class below ensures that the side length of its triangle is always the same as the side length of its square."
+        "If you don’t need to compute the property but still need to provide code that is run before and after setting a new value, use `willSet` and `didSet`. `willSet` is called just before the value is stored. `didSet` is called immediately after the new value is stored. The code you provide is run any time the value changes outside of an initializer. For example, the class below ensures that the side length of its triangle is always the same as the side length of its square."
```

Comment:
I'm not sure how common `willSet` and `didSet` are outside of Swift, but I was only first introduced to them when I was working on iOS apps at my second internship since it's often used in an iOS app's `UIView` or `UIViewController`. So to anyone not familiar with these two, I thought I should add the explanation (even if the name itself is a good hint!)